src/source/JavaGenerator.scala

```diff
@@ -340,6 +340,19 @@ class JavaGenerator(spec: Spec) extends Generator(spec) {
 
         }
 
+        w.wl
+        w.wl("@Override")
+        w.w("public String toString()").braced {
+          w.w(s"return ").nestedN(2) {
+            w.wl(s"\042${self}{\042 +")
+            for (f <- r.fields) {
+              w.wl(s"\042,\042 + \042${idJava.field(f.ident)}=\042 + ${idJava.field(f.ident)} +")
+            }
+          }
+          w.wl(s"\042}\042;")
+        }
```

Comment:
I tried that initially, but had troubles with it. Here's what happens when I search/replace for `\042` into `"""`

```
[info] Compiling 4 Scala sources to /Users/steipete/Documents/Projects/djinni/src/target/scala-2.11/classes...
[error] /Users/steipete/Documents/Projects/djinni/src/source/JavaGenerator.scala:347: unclosed string literal
[error]             w.wl(s""""${self}{""" +")
[error]                                    ^
[error] /Users/steipete/Documents/Projects/djinni/src/source/JavaGenerator.scala:349: unclosed string literal
[error]               w.wl(s"""",""" + """${idJava.field(f.ident)}=""" + ${idJava.field(f.ident)} +")
[error]                                                                                            ^
[error] /Users/steipete/Documents/Projects/djinni/src/source/JavaGenerator.scala:352: unclosed string literal
[error]           w.wl(s""""}""";")
[error]                          ^
[error] /Users/steipete/Documents/Projects/djinni/src/source/JavaGenerator.scala:352: ')' expected but ';' found.
[error]           w.wl(s""""}""";")
```